README.md

```diff
@@ -35,23 +36,27 @@ Playwright can be used to create a browser instance, open pages, and then manipu
 
 #### Page screenshot
 
-This code snippet navigates to example.com in WebKit, and saves a screenshot.
+This code snippet navigates to whatsmyuseragent.org in Chromium, Firefox and WebKit, and saves 3 screenshots.
 
 ```js
 const pw = require('playwright');
 
 (async () => {
-  const browser = await pw.webkit.launch(); // or 'chromium', 'firefox'
-  const context = await browser.newContext();
-  const page = await context.newPage();
+  const browsers = ['chromium', 'firefox', 'webkit'];
 
-  await page.goto('https://www.example.com/');
-  await page.screenshot({ path: 'example.png' });
+  browsers.forEach(async browserName => {

```

Comment:
Let's use modern api and also not run them in parallel:

for (const browser of ['chromium', 'firefox', 'webkit']) {
   ...
}